apps/frontend/src/app/components/map-testing-request-selection/map-testing-request-selection.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, forwardRef, ViewChild } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { MAX_TESTING_REQUESTS, User } from '@momentum/constants';
+import { NgClass } from '@angular/common';
+import { UserSearchComponent } from '../search/user-search/user-search.component';
+import { TooltipDirective } from '../../directives';
+import { AvatarComponent } from '../avatar/avatar.component';
+
+@Component({
+  selector: 'm-map-testing-request-selection',
+  templateUrl: 'map-testing-request-selection.component.html',
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => MapTestingRequestSelectionComponent),
+      multi: true
+    }
+  ],
+  standalone: true,
+  imports: [NgClass, UserSearchComponent, TooltipDirective, AvatarComponent]
+})
+export class MapTestingRequestSelectionComponent
+  implements ControlValueAccessor
+{
+  protected readonly MAX_TESTING_REQUESTS = MAX_TESTING_REQUESTS;
+
+  protected users: User[] = [];
+  protected readonly max = MAX_TESTING_REQUESTS;
```

Comment:
This is probably redundant, cause MAX_TESTING_REQUESTS there is also a protected readonly value, which also used in template